src/components/ApproveRequests.vue

```diff
@@ -1,6 +1,13 @@
+/* eslint-disable vue/no-use-v-if-with-v-for */
 <template>
-    <ul class="campaigns" v-for="campaign in userContributions" v-bind:key="campaign.id">
+  <Loading v-if="loading"/>
+    <ul
```

Comment:
Spread the ul out, set loading above it so we don't get a list of loading screens.